src/styles/blocks/productCard/stars.scss

```diff
@@ -0,0 +1,15 @@
+.product__rating-stars {
+  display: flex;
+  justify-content: space-between;
+
+  &-star {
+    background-image: url(../images/star.svg);
+    background-position: 50%;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  @include stars($starsAmount);
```

Comment:
The mixin `@include stars($starsAmount);` is used here, but it is not defined within this file. Ensure that the mixin `stars` is defined elsewhere or imported correctly to avoid compilation errors.